src/Mod/PartDesign/Gui/ViewProviderDatumPlane.cpp

```diff
@@ -45,11 +46,31 @@ ViewProviderDatumPlane::ViewProviderDatumPlane()
 
     pCoords = new SoCoordinate3();
     pCoords->ref ();
+
+    // setup semi transparent face material
+    const App::Color &c = ShapeColor.getValue();
+    pPlaneFaceMaterial = new SoMaterial();
+    pPlaneFaceMaterial->transparency.setValue(0.95f);

```

Comment:
#9391 says this PR isn't acceptable because of this hardcoded value. So is this PR to me moved to draft ?

Reply:
Yes, sorry I forgot to make this a draft